Scripts/capture-build-details.sh

```diff
@@ -64,13 +64,25 @@ info "Gathering build details in ${git_source_root}"
 cd "${git_source_root}"
 
 if [ -e .git ]; then
-  rev=$(git rev-parse HEAD)
-  plutil -replace com-loopkit-Loop-git-revision -string ${rev} "${info_plist_path}"
+  # if branch exists, report Loop branch and SHA
+  # else, go up one level and report LoopWorkspace values
   branch=$(git branch --show-current)
   if [ -n "$branch" ]; then
-    plutil -replace com-loopkit-Loop-git-branch -string "${branch}" "${info_plist_path}"
+    rev=$(git rev-parse HEAD)
+    plutil -replace com-loopkit-Loop-git-revision -string "Loop ${rev:0:7}" "${info_plist_path}"
```

Comment:
The git-revision shouldn't have "Loop" in it, nor should the git-branch; app name is taken from other parts of the plist.

Reply:
You could add a 'com-loopkit-Loop-build-type' pref if you wanted. But I'm not exactly sure if "build type" is what you're trying to track.

Reply:
Yes, both those changes sound good. It might be cleaner to just not add those plist values for a non-workspace build.